pkg/chunked/cache_linux.go

```diff
@@ -559,22 +559,22 @@ func unmarshalToc(manifest []byte) (*internal.TOC, error) {
 		for iter.ReadArray() {
 			var m internal.FileMetadata
 			for field := iter.ReadObject(); field != ""; field = iter.ReadObject() {
-				switch field {
+				switch strings.ToLower(field) {
```

Comment:
Why does this location switch to treating field names returned by `iter.ReadObject()` in a case-insensitive fashion while the previous loop over the same values doesn't?

Reply:
thanks for spotting it.  I've changed also the previous loop to use the case-insensitive values.